### ISSUE TITLE
Documentation issue: Theme application consistency

### ISSUE THREAD

**Issue by jsrikrishna11** [siddharthkp/react-ui#91]
After choosing a theme other than the default theme which is `'light'`. If I refresh the page, the page theme is being reset to `'light'`. This is because, on reload it is calling `useState('light')`again.

Is it even an issue?

**Comment by jsrikrishna11** [comment]
I'd like to work on this issues. Is that okay?

**Comment by rubenmoya** [comment]
Sure, go for it!

**Comment by iamdarshshah** [comment]
Yes, We can store the current theme in local-storage (when a user visits the website it will be set to`light` by default) and we can update it as per the user convenience. 

Thus, When a user visits the site theme will be set as per the user's selected theme (by default it will be set to `light`).

**Comment by jsrikrishna11** [comment]
I'll try and work on this issue, unless someone has already taken it up.
Let me know :)

**Comment by jsrikrishna11** [comment]
I would like to work on it, unless some else took it up. Let me know :)

**Comment by siddharthkp** [comment]
@jsrikrishna11 Go for it :)
